common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -570,6 +569,20 @@
                                 gettext('Show posts by {username}.'), {username: username}
                             );
                             self.addSearchAlert(message, 'search-by-user');
+                            DiscussionUtil.safeAjax({
+                                url: DiscussionUtil.urlFor('followed_threads', response.users[0].id),
```

Comment:
So I'm the person who submitted the ticket. The problem isn't that the discussion posts aren't showing up in search results; the problem is that *the discussion posts aren't showing up **at all***. Anywhere. Even when I view all posts for a particular topic, discussion posts are missing from each topic I've looked at. I hope these posts weren't permanently deleted somehow, because some of them reported bugs in the course or other things that need attention from the course team.